mandatory/2-function-creation.js

```diff
@@ -15,26 +22,58 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  const largeDiscountThreshold = 200;
+  const largeDiscountPerc = 0.9;
+  const smallDiscountPerc = 0.95;
+
+  if (totalPrice >= largeDiscountThreshold) {
+    return totalPrice * largeDiscountPerc;
+  } else {
+    return totalPrice * smallDiscountPerc;
+  }
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  let i = 1;
+  while (i <= limit) {
```

Comment:
Thought: I think this is OK, but one reading could be that it should log numbers between 1 and limit (non inclusive of limit). in that case you would want to do `while (i < limit)`